File: legacy/isma-tools/src/main/java/ru/nstu/isma/print/MatrixPrint.java
package ru.nstu.isma.print;

/**
 * Created by Bessonov Alex
 * on 14.03.2015.
 */
public class MatrixPrint {
    public static void print(String label, double[][] matrix, double[] b) {
        System.out.println(" =====  Matrix: " + label + " ========");
        for (int i = 0; i < matrix.length; i++) {
            for (int j = 0; j < matrix[i].length; j++) {
                System.out.print(matrix[i][j] + "   \t");
            }
            System.out.print("\t   =   \t");
            System.out.println(b[i]);
        }
    }

    public static void print(String label, double[][] matrix) {
        System.out.println(" =====  Matrix: " + label + " ========");
        for (int i = 0; i < matrix.length; i++) {
            for (int j = 0; j < matrix[i].length; j++) {
                System.out.print(matrix[i][j] + "   \t");
            }
            System.out.print("\n");
        }
    }
}

File: isma-next-tools/src/main/kotlin/ru/nstu/isma/print/MatrixPrint.kt
package ru.nstu.isma.print

/**
 * Created by Bessonov Alex
 * on 14.03.2015.
 */
object MatrixPrint {
    @JvmStatic
    fun print(label: String, matrix: Array<DoubleArray>, b: DoubleArray) {
        println(" =====  Matrix: $label ========")
        for (i in matrix.indices) {
            for (element in matrix[i]) {
                print("$element   \t")
            }
            print("\t   =   \t")
            println(b[i])
        }
    }

    @JvmStatic
    fun print(label: String, matrix: Array<DoubleArray>) {
        println(" =====  Matrix: $label ========")
        for (i in matrix.indices) {
            for (element in matrix[i]) {
                print("$element   \t")
            }
            print("\n")
        }
    }
}